### DISCUSSION TITLE
Change the backdrop of modal presentation.

### DISCUSSION THREAD

**Discussion by AkshayAdiga1997** [software-mansion/react-native-screens#807] [Q&A] [answered]
I am using _createNativeStackNavigator_
with the config
```
headerShown: false,
stackPresentation: 'modal',
gestureEnabled: false,
```
I have set colors.background to #121212 (NavigationContainer theme)
Is there a way to change the opacity or set the background to somewhere near black? Currently the view **behind the moda**l turns **greyish** which looks odd. Thanks.
https://user-images.githubusercontent.com/19915310/107257885-29c3c580-6a61-11eb-89fd-01c225352a63.MP4

**Comment by WoLewicki** [comment, marked as answer]
It is currently not customizable, but we are open to PRs anytime 😅

**Reply by AkshayAdiga1997** [reply]
> It is currently not customizable, but we are open to PRs anytime 😅

Okay thanks.
